src/Sylius/Bundle/AdminBundle/Console/Command/ListAdminUsersCommand.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Console\Command;
+
+use Sylius\Component\Core\Model\AdminUserInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\TableStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(
+    name: 'sylius:admin-user:list',
+    description: 'Displays all admin users with information in a list. Optionally, also one user can be displayed if the username is specified as an option.',
+)]
+final class ListAdminUsersCommand extends Command
+{
+    protected SymfonyStyle $io;
+
+    /** @param UserRepositoryInterface<AdminUserInterface> $userRepository */
+    public function __construct(
+        private readonly UserRepositoryInterface $userRepository,
+    ) {
+        parent::__construct();
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output): void
+    {
+        $this->io = new SymfonyStyle($input, $output);
+    }
+
+    protected function configure(): void
+    {
+        $this->setAliases(['sylius:admin-users:list']);
+        $this->addOption('username', null, InputOption::VALUE_REQUIRED, 'List one user by the given username');
+    }
```

Comment:
```suggestion
    protected function configure(): void
      {
          $this->setAliases(['sylius:admin-users:list']);
          $this->addOption(
              'search',
              's',
              InputOption::VALUE_REQUIRED,
              'Search users by email, username, first name or last name (partial match)',
          );
      }
```
It would be more useful to allow partial, more flexible matching, for instance:
```
  sylius:admin-user:list -s sylius          # username
  sylius:admin-user:list -s @example        # email domain
  sylius:admin-user:list -s john            # first/last name
  sylius:admin-user:list -s doe             # last name
  sylius:admin-user:list                    # all
```

It also could be case-insensitive:

<img width="980" height="409" alt="Image" src="https://github.com/user-attachments/assets/d34d9b38-aec1-4933-9341-23371bd80e16" />

Reply:
I like the idea! :) I tested it out and its working great.